doc/whatsnew/fragments/5920.other

```diff
@@ -0,0 +1,3 @@
+Pylint now support Python 3.11.
```

Comment:
```suggestion
Pylint now provides basic support for Python 3.11.
```

Should we also update the classifier?

Reply:
Marc made an argument not to in astroid. When someone search for the meta on pypi they expect to have really compatible package. pip will find a version compatible on 3.11, I think right now this is the goal.